pages/in/getapp.vue

```diff
@@ -75,7 +75,7 @@ export default {
   }
 
   &__download-button {
-    margin: 20px;
+    margin: 20px 20px 10px 20px;
```

Comment:
Please change to padding

Reply:
should be margin, this is to decrease size for padding